### ISSUE TITLE
[Sourcing][Accessibility][Vertical Navigation]No label and state of the parent node read in list, screenreader simply reads button

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10388]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
No label and the node's state (whether expanded or collapsed ) is read for parent node in list, screenreader simply reads button

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
replicable on https://fundamental-ngx.netlify.app/#/core/vertical-navigation

#### If this is a bug, please provide steps for reproducing it.

https://github.com/SAP/fundamental-ngx/assets/88182358/a6cac087-5b9a-4f3b-9288-105e6ce9ec55

**Comment by nikolay-kolarov** [comment]
Hi,

I guess the button has been reused for convenience. Still, please have a look and use the UI5 implementation as a reference: https://sapui5.hana.ondemand.com/#/entity/sap.tnt.ToolPage/sample/sap.tnt.sample.ToolPage
The focus goes on the whole item and the button is hidden from the screen reader (has role presentation).

Best Regards,
Nikolay

**Comment by droshev** [comment]
fixed with `0.43.9`

**Comment by sagar7259** [comment]
Hello, 
The reported issue fixed has now there is no separate tab stop provided for the button. This ticket can be closed.

Regards,
Sagar (ATL team)
